InfiniteRecharge/src/main/java/frc/robot/config/Config.java

```diff
@@ -14,10 +14,10 @@
     public int INTAKE_MOTOR_ID = 14;
 
     // Drive
-    public int LEFT_DRIVE_LEADER_ID = 15;
-    public int LEFT_DRIVE_FOLLOWER_ID = 2;
-    public int RIGHT_DRIVE_LEADER_ID = 3;
-    public int RIGHT_DRIVE_FOLLOWER_ID = 4;
+    public int LEFT_DRIVE_LEADER_ID = 1;

```

Comment:
For prototyping, the azimuth motor is currently configured as motor 1.

JuniorConfiguration already sets the motor ids for this